.github/workflows/Unit-Test.yml

```diff
@@ -19,4 +19,8 @@ jobs:
             CREW_REPO=${{ github.event.pull_request.head.repo.clone_url }} CREW_BRANCH=${{ github.head_ref }} crew update && \
             yes | crew upgrade && \
             yes | crew install vim && \
-            yes | crew remove vim"
+            yes | crew remove vim && \
+            git sparse-checkout add ../tests && \
```

Comment:
This sparse checkout of tests should no longer be needed, as it is added from `install.sh`.

Reply:
ok, this line should really not be needed after #8994 is merged and I rebuild containers! I will ping when that is done...

Reply:
Ok, this should be good to be removed. Both crew and install.sh are now adding the `tests` dir...